monai/transforms/utility/dictionary.py

```diff
@@ -1424,20 +1437,16 @@ class RandTorchVisiond(RandomizableTransform, MapTransform):
     backend = TorchVision.backend
 
     def __init__(
-        self, keys: KeysCollection, name: str, prob: float = 1.0, allow_missing_keys: bool = False, *args, **kwargs
+        self,
+        keys: KeysCollection,
+        name: str,
+        apply_prob: float = 1.0,
```

Comment:
I'm not sure why changing the `prob` to `apply_prob`? currently only the RandCuCIM has this parameter and looks like a typo to me, to be consistent with the rest of the transforms they should be `prob`. and I think the change should also have a deprecating warning message.

Reply:
Wenqi, it is different than `prob` in other transformer as the true probability comes from another parameter `p` or `prob` of the underlying transform. We should distinguish this two. As an input argument of a call, we can have `apply_probe=1.0, prob=0.5`.

Reply:
As `apply_prob` is just reserve for [not-very-usual] specific use cases.

Reply:
I see, in cucim there's the same `prob`, in my opinion we should pass `cucim_prob` to cucim's prob api.  torchvision seems to use `p` to denote this value

Reply:
Consider following examples:

This always applies `rand_image_flip` (apply_probe=1.0) but `rand_image_flip` will flip it randomly (p=0.5)
```python
RandCuCIM(name="rand_image_flip", apply_probe=1.0, prob=0.5, spatial_axis=-1),
```

This applies `image_flip` randomly (apply_probe=0.5) but `image_flip` is not a random transform so always being applied.
```python
RandCuCIM(name="rand_image_flip", apply_probe=0.5, spatial_axis=-1),
```

This applies `image_flip` randomly (apply_probe=0.5) and `rand_image_flip` will flip it randomly (p=0.5). Overall randomness of 0.25.
```python
RandCuCIM(name="rand_image_flip", apply_probe=0.5, prob=0.5, spatial_axis=-1),
```

Reply:
thanks for the example, I don't find the same for torchvision. Also in my opinion we should use `RandCuCIM(name="rand_image_flip", prob=1.0, cucim_prob=0.5, spatial_axis=-1)` to distinguish 3rd party params

Reply:
`apply_prob` is a rarely used option so we should not make it `prob` (either in cucim or torchvision). We want to have the same API for the user for the typical use of this transform is:
```python
RandCuCIM(name="rand_image_flip", prob=0.5, spatial_axis=-1)
```
similar to other transforms, and it should be the same for torchvision.

Consider this:
```python
RandCuCIM(name="RandomHorizontalFlip", prob=0.5, spatial_axis=-1)
```
This seems to be fine but user think it is applying it with probability of 0.5 but actually it is 0.25 because we have `prob=0.0` and the default for `RandomHorizontalFlip` which is `p=0.5`, so confusing!

Maybe we should map `prob` to the `p` for torchvision transforms but not the other way around.

Reply:
@ericspod @Nic-Ma @rijobro may I know your opinion here? thanks

Reply:
Hi @drbeh , @wyli ,

Sorry for the late response, I think through this problem here, I personally feel we should not inherit from `RandomizableTransform` because for most cases we actually don't need this `apply_prob`(or call it `prob`) arg. If we really want to set apply prob for a non-random transform, we can consider to enhance the `RandomizableTransform` transform or make a subclass of it to be a wrapper of some other non-random transforms: `my_trans = RandomizableTransform(transform=Flip(...), prob=0.5)`. It also follows the "Composition Over Inheritance" design principle.
What do you think?

Thanks.

Reply:
@Nic-Ma, what you mentioned is a great point. The source of confusion is this over-inheritance! If someone wants to randomly apply a random (or non-random) transform, she should be explicit about it and use composition instead of inherence.

@wyli @SachidanandAlle Here is my suggestion. Please let me know what do you think.
(Below I removed the `key=...` for all the transforms for simplicity but assume that it is there.)

| application mode | transform type | torchivison wrapper | example
| --- | --- | --- | ---
| non-random | non-random | `TorchVisiond` | `TorchVisiond(name="CenterCrop")`
| non-random | random | `RandTorchVisiond` | `RandTorchVisiond(name="RandomHorizontalFlip", p=0.5)`
| random | non-random | `RandomApplyd` | `RandomApplyd(transform=TorchVisiond(name="CenterCrop"), prob=0.5)`
| random | random | `RandomApplyd` | `RandomApplyd (transform=RandTorchVisiond(name="RandomHorizontalFlip", p=0.5), prob=0.5)`


Here instead of `RandTorchVisiond`, `RandomApplyd` will inherit from `RandomizableTransform`, and can be used for any MONAI transform, including `TorchVision`, `RandTorchVision`, `CuCIM`, and `RandCUCIM`.

Reply:
there's already a `RandLambdaD` in monai

Reply:
Sure, that would work too. Then we can remove the `RandomizebleTransform` inherence from `RandTorchVision` and `RandCUCIM`, and `prob` or `apply_prob` arguments. Sounds good?

Reply:
not really... the cache-based datasets relies on it to determine caching

probably `RandomizableTrait` should be used
 https://github.com/Project-MONAI/MONAI/blob/d0db5fd9d4da3bc7027d22c52bb9c405c3b9e879/monai/transforms/transform.py#L160

Reply:
I know, avoid caching is the whole point of this transforms. I used `Randomizable` but I will change it to `RandomizableTrait`.

Reply:
@drbeh I feel this proposal looks good to me, `RandLambdaD` should work for the case.
Let's try to make the classes clear and independent.

Thanks.

Reply:
sure, if we go for this approach we should review the usage of `Randomizable`, `RandomizableTrait`, `RandomizableTransform`, such as:

- https://github.com/Project-MONAI/MONAI/blob/1516ca758090dd8ed2890b64554a77f681a0e224/monai/transforms/nvtx.py#L66
- https://github.com/Project-MONAI/MONAI/blob/025c10751a1be596b3be9d0b64bb095e653804c3/monai/data/dataset.py#L280

Reply:
some test errors
```
[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_0 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: color_jitter() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_1 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: image_flip() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_2 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_3 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_4 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: scale_intensity_range() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_5 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_6 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: rand_zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.968Z] ======================================================================

[2022-12-05T22:47:20.968Z] ERROR: test_tramsforms_cupy_batch_7 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.968Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.968Z] Traceback (most recent call last):

[2022-12-05T22:47:20.968Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.968Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 169, in test_tramsforms_cupy_batch

[2022-12-05T22:47:20.968Z]     output = RandCuCIMd(keys="image", apply_prob=0.0, **params)(input)["image"]

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.968Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.968Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.968Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.968Z] TypeError: rand_zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.968Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_0 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: color_jitter() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_1 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: image_flip() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_2 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_3 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_4 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: scale_intensity_range() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_5 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_6 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: rand_zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_cupy_single_7 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 141, in test_tramsforms_cupy_single

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: rand_zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_numpy_batch_0 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: color_jitter() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_numpy_batch_1 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.969Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.969Z] Traceback (most recent call last):

[2022-12-05T22:47:20.969Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.969Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.969Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.969Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.969Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.969Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.969Z] TypeError: image_flip() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.969Z] 

[2022-12-05T22:47:20.969Z] ======================================================================

[2022-12-05T22:47:20.969Z] ERROR: test_tramsforms_numpy_batch_2 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_batch_3 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_batch_4 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: scale_intensity_range() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_batch_5 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_batch_6 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: rand_zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_batch_7 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 117, in test_tramsforms_numpy_batch

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: rand_zoom() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_single_0 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 93, in test_tramsforms_numpy_single

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: color_jitter() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_single_1 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 93, in test_tramsforms_numpy_single

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: image_flip() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_single_2 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 93, in test_tramsforms_numpy_single

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_single_3 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.970Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.970Z] Traceback (most recent call last):

[2022-12-05T22:47:20.970Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.970Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 93, in test_tramsforms_numpy_single

[2022-12-05T22:47:20.970Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.970Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.970Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.970Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.970Z] TypeError: rand_image_rotate_90() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.970Z] 

[2022-12-05T22:47:20.970Z] ======================================================================

[2022-12-05T22:47:20.970Z] ERROR: test_tramsforms_numpy_single_4 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.971Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.971Z] Traceback (most recent call last):

[2022-12-05T22:47:20.971Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.971Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.971Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 93, in test_tramsforms_numpy_single

[2022-12-05T22:47:20.971Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.971Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.971Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.971Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.971Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.971Z] TypeError: scale_intensity_range() got an unexpected keyword argument 'apply_prob'

[2022-12-05T22:47:20.971Z] 

[2022-12-05T22:47:20.971Z] ======================================================================

[2022-12-05T22:47:20.971Z] ERROR: test_tramsforms_numpy_single_5 (tests.test_rand_cucim_dict_transform.TestRandCuCIMDict)

[2022-12-05T22:47:20.971Z] ----------------------------------------------------------------------

[2022-12-05T22:47:20.971Z] Traceback (most recent call last):

[2022-12-05T22:47:20.971Z]   File "/opt/conda/lib/python3.8/site-packages/parameterized/parameterized.py", line 533, in standalone_func

[2022-12-05T22:47:20.971Z]     return func(*(a + p.args), **p.kwargs)

[2022-12-05T22:47:20.971Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_rand_cucim_dict_transform.py", line 93, in test_tramsforms_numpy_single

[2022-12-05T22:47:20.971Z]     output = RandCuCIMd(keys="image", apply_prob=1.0, **params)(input)["image"]

[2022-12-05T22:47:20.971Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/dictionary.py", line 1704, in __call__

[2022-12-05T22:47:20.971Z]     d[key] = self.trans(d[key])

[2022-12-05T22:47:20.971Z]   File "/home/jenkins/agent/workspace/MONAI-premerge/monai/monai/transforms/utility/array.py", line 1361, in __call__

[2022-12-05T22:47:20.971Z]     return self.transform(data, *self.args, **self.kwargs)

[2022-12-05T22:47:20.971Z] TypeError: zoom() got an unexpected keyword argument 'apply_prob'
```